src/subapps/admin/components/ViewForm/SparqlQueryInput.tsx

```diff
@@ -1,25 +1,30 @@
-import React from 'react';
+import React, { useRef } from 'react';
+import * as codemirror from 'codemirror';
+import { UnControlled as UnControlledCodeMirror } from 'react-codemirror2';
```

Comment:
General understanding question: Why do we need codemirrror and react-codemirror2?

Reply:
`react-codemirror2` require `codemirror`, we can use only codemirror, just was used long time before and also in the same time it's very helpful to do all the config abstraction